src/pages/using-spark/foundations/color-usage.mdx

```diff
@@ -268,8 +268,8 @@ import {
       Input
     </label>
     <input
+      class="sprk-b-TextInput"
```

Comment:
The gatsby site would have to be updated to get this styling change. So removing the 100% like this would break it.